norfair/drawing.py

```diff
@@ -41,6 +48,20 @@ def draw_points(
                 thickness=thickness,
             )
 
+        if draw_labels:
+            label_draw_position = np.array([min(points[:, 0]), min(points[:, 1])])
+            label_draw_position -= radius
+            cv2.putText(
+                frame,
+                f"Label: {d.label}",
```

Comment:
"Label" string may occupy too much space when there are many objects on the image, it may make sense to make it shorter, or optional.